tox.ini

```diff
@@ -6,7 +6,7 @@ platform = linux: linux
            win: win64
 
 [flake8]
-ignore = E12, W503
+ignore = E12, W503, E722, E226, E302, E41, E228, E225, W291, E501, F841, E303, W293, W391
```

Comment:
bad idea: please revert this change and fix all PEP8 issues in code

Reply:
Fixed